includes/admin/class-wc-payments-admin.php

```diff
@@ -287,11 +289,11 @@ public function register_payments_scripts() {
 			'WCPAY_ADMIN_SETTINGS',
 			'wcpayAdminSettings',
 			[
-				'accountStatus'           => $this->account->get_account_status_data(),
-				'accountFees'             => $this->account->get_fees(),
 				'fraudServices'           => $this->account->get_fraud_services_config(),
 				// TODO: Remove this line ahead of releasing Apple Pay for all merchants.
 				'paymentRequestAvailable' => WC_Payments::should_payment_request_be_available(),
+				'accountStatus'           => $this->account->get_account_status_data(),
+				'accountFees'             => $this->account->get_fees(),
```

Comment:
Not sure if that was intentional, but as I understand this change moves the lines down to the list. Do we need to keep it?